config/environments/test.rb

```diff
@@ -40,4 +40,7 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Make sure there is no cache
+  config.cache_store = :null_store
```

Comment:
Ah yeah, good call - one potential solution is to store the snap prices in the same Redis instance as the Sidekiq jobs, since that Redis instance must have persistence, in which case this configuration would still be okay, but I will remove it for now.

Reply:
I think ideally tests wouldn't rely on cache since it could cause issues. But I think given the current architecture it may not yet be a possibility. :disappointed: